### PR TITLE
#18: Add github actions

### PR DESCRIPTION
**Pull request by pbrudnick**

### CODE REVIEW

File: .github/workflows/erlang.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      image: erlang:22.0.7
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Compile
+      run: rebar3 compile
+    - name: Format check
+      run: rebar3 format --verify
+    - name: Run tests
+      run: rebar3 do eunit, ct
```
Comment (elbrujohalcon) [inline .github/workflows/erlang.yml:25]: We have a `test` target for this.
```suggestion
      run: rebar3 test
```

**Review by elbrujohalcon — CHANGES_REQUESTED** [review]

File: .github/workflows/erlang.yml (hunk 1 above)
Comment (pbrudnick) [reply .github/workflows/erlang.yml:25]: yes but it outputs coverage, np?

**Comment by pbrudnick** [comment]
The actions are running already here, and failing because `noent` on tests, fixed in https://github.com/AdRoll/rebar3_hank/pull/22

File: .github/workflows/erlang.yml (hunk 1 above)
Comment (elbrujohalcon) [reply .github/workflows/erlang.yml:25]: no problem… if GA eventually understands how to interpret and check that… that would be even better! To ensure that we don't reduce coverage, I mean.

**Review by elbrujohalcon — APPROVED** [review]

**Pull request merged**